index.html

```diff
@@ -690,164 +434,95 @@ <h2>Attributes</h2>
-      For a browser implementing ORTC, a <code><a>RTCQuicTransport</a></code>
-      object assumes a QUIC role of <code>auto</code> upon construction.
-      This implies that the QUIC role is determined by the ICE role.  Since
-      <code>getLocalParameters().role</code> always returns the role assigned
-      to an <code><a>RTCQuicTransport</a></code> object upon construction
-      (<code>auto</code> for a browser), the <code>getLocalParameters</code>
-      method cannot be used to determine the desired or actual role of an
-      <code><a>RTCQuicTransport</a></code>.</p>
-      <p>An application can determine the
-      desired role of an <code><a>RTCQuicTransport</a></code> from the value of
-      <code><var>remoteParameters</var>.role</code> passed to
-      <code><a>RTCQuicTransport</a>.start(<var>remoteParameters</var>)</code>.
-      If <code><var>remoteParameters</var>.role</code> is <code>server</code>
-      then the desired role of the <code><a>RTCQuicTransport</a></code>
-      is <code>client</code>. If <code><var>remoteParameters</var>.role</code>
-      is <code>client</code> then the desired role of the
-      <code><a>RTCQuicTransport</a></code> is <code>server</code>.</p>
-      <p>The <code>RTCQuicTransport.transport.onstatechange</code> EventHandler
-      can be used to determine whether an <code><a>RTCQuicTransport</a></code>
-      transitions to the desired role. When
-      <code><a>RTCQuicTransport</a>.transport.state</code> transitions to
-      <code>connected</code>, if <code><a>RTCQuicTransport</a>.transport.role</code>
-      is <code>controlled</code> then the role of the
-      <code><a>RTCQuicTransport</a></code> is <code>client</code>.
-      If <code><a>RTCQuicTransport</a>.transport.role</code>
-      is <code>controlling</code> then the role of the
-      <code><a>RTCQuicTransport</a></code> is <code>server</code>.</p>
-    </section>
-    <section id="rtcquictransportstate*">
-      <h3><dfn>RTCQuicTransportState</dfn> Enum</h3>
-      <p><code>RTCQuicTransportState</code> indicates the state of the QUIC
+    <section id="quictransportstate*">
+      <h3><dfn>QuicTransportState</dfn> Enum</h3>
+      <p><code>QuicTransportState</code> indicates the state of the QUIC
       transport.</p>
       <div>
-        <pre class="idl">enum RTCQuicTransportState {
+        <pre class="idl">enum QuicTransportState {
     "new",
     "connecting",
     "connected",
     "closed",
     "failed"
 };</pre>
-        <table data-link-for="RTCQuicTransportState" data-dfn-for="RTCQuicTransportState"
+        <table data-link-for="QuicTransportState" data-dfn-for="QuicTransportState"
         class="simple">
           <tbody>
             <tr>
               <th colspan="2">Enumeration description</th>
             </tr>
             <tr>
-              <td><dfn><code id="idl-def-RTCQuicTransportState.new">new</code></dfn></td>
+              <td><dfn><code id="idl-def-QuicTransportState.new">new</code></dfn></td>

```

Comment:
Does it matter that this state doesn't apply to the QuicTransport? It only is relevant for the RTCQuicTransport.

Reply:
I don't think that matters.